### PR TITLE
Update changelog for 0.30.0 release

### PR DESCRIPTION
**Pull request by sklam**
and fix README.rst for proper rendering in PyPI (checked with `python setup.py check -r`)

### CODE REVIEW

**Comment by pitrou** [comment]
Looks fine to me.

File: CHANGE_LOG [hunk 1]
```diff
@@ -1,3 +1,35 @@
+Version 0.30.0
+---------------
+
+This release adds preliminary support for Python3.6, but no official build is
```
Comment (stuartarchibald) [inline CHANGE_LOG]: Space before 3.6 please.

File: CHANGE_LOG [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+available yet.  A new system reporting tool (``numba --sysinfo``) is added to
+provide system information to help core developers in replication and debugging.
+See below for other improvements and bug fixes.
+
+Improvements:
+
+* PR #2209: Support Python3.6.
```
Comment (stuartarchibald) [inline CHANGE_LOG]: Space before 3.6 please.

File: CHANGE_LOG [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
+* PR #2206, Issue #2163: Fix ``zip()`` consuming extra elements in early
+  exhaustion.
+* PR #2185, Issue #2159, #2169: Fix rewrite pass affecting objmode fallback.
+* PR #2204, Issue #2178: Fix annotation for liftedloop.
+* PR #2203: Fix Appveyor segfault with Python3.5.
+* PR #2202, Issue #2198: Fix target context not initialized when loading from
+  ufunc cache.
+* PR #2172, Issue #2171: Fix optional type unpacking.
+* PR #2189, Issue #2188: Disable freezing of big (>1MB) global arrays.
+* PR #2189, Issue #2179: Fix invalid variable version in looplifting.
```
Comment (stuartarchibald) [inline CHANGE_LOG]: There's 2x #2189, think the lower one should be #2180

File: CHANGE_LOG [hunk 4]
```diff
@@ -1,3 +1,35 @@
...
+* PR #2186: Add system reporting tool.
+
+Fixes:
+
+* PR #2214, Issue #2212: Fix memory error with ndenumerate and flat iterators.
+* PR #2206, Issue #2163: Fix ``zip()`` consuming extra elements in early
+  exhaustion.
+* PR #2185, Issue #2159, #2169: Fix rewrite pass affecting objmode fallback.
+* PR #2204, Issue #2178: Fix annotation for liftedloop.
+* PR #2203: Fix Appveyor segfault with Python3.5.
```
Comment (stuartarchibald) [inline CHANGE_LOG]: Space before 3.5 please.

**Comment by MSeifert04** [comment]
Am I right that this fixes #2211?

**Comment by sklam** [comment]
> Am I right that this fixes #2211?

It should.

**Comment by stuartarchibald** [comment]
Thanks for the fixup. Looks good.

**Pull request merged**